src/components/Reporting.js

```diff
@@ -0,0 +1,54 @@
+import { Component, PropTypes, DOM as dom, createFactory } from 'react';
+import { connect } from 'react-redux';
+import actions from '../action_creators';
+import ReportingNavComponent from './ReportingNav';
+
+const ReportingNav = createFactory(ReportingNavComponent);
+
+function mapStateToProps(state) {
+  return {
+    orders: state.orders
+  };
+}
+
+class Reporting extends Component {
+  constructor(props) {
+    super(props);
+    this.setActiveTab = this.setActiveTab.bind(this);
```

Comment:
Why define setActiveTab and tracking activeTab in the ReportComponent instead of the ReportingNavComponent?